configuration.go

```diff
@@ -19,8 +27,15 @@ type Configuration struct {
 	MaxIdleConnsPerHost   int
 }
 
-func NewConfiguration() *Configuration {
+func NewConfiguration(addrList, basePath, user string, https bool) *Configuration {
```

Comment:
Add comment about the format for address list (semi-colon separated)
添加关于地址列表格式的注释（分号分隔）

Reply:
I would suggest to keep function `NewConfiguration()` and create a new function called `NewMultiAddrConfiguration()`. This will support backward compatibility of people already using this API.

我建议保留函数“NewConfiguration()”并创建一个名为“NewMultiAddrConfiguration()”的新函数。 这将支持已经使用此 API 的人的向后兼容性。